students/kulon_aleksander/lesson_05/greater_than_neighbours.py

```diff
@@ -0,0 +1,16 @@
+def func(arg):
+    result = ''
+    return result
+
+
+def main():
+    func(1)
+    a = "-9 29 -100 64 26 73 -96 28 -92 11 -14 -86 -54 -67".split()
+    a = input().split()
+    for i in range(len(a)):
```

Comment:
In this task you should print the quantity of elements that are greater than both of their neighbors not each element. Please change it.